src/js/setup-text-tracks.js

```diff
@@ -0,0 +1,136 @@
+import dashjs from 'dashjs';
+
+/*
+ * Attach text tracks from dash.js to videojs
+ *
+ * @param {videojs} player the videojs player instance
+ * @param {array} tracks the tracks loaded by dash.js to attach to videojs
+ *
+ * @private
+ */
+function attachDashTextTracksToVideojs(player, tracks) {
+  const mediaPlayer = player.dash.mediaPlayer;
+
+  const trackDictionaryKeyedByTextTrack = new WeakMap();
```

Comment:
My mistake, I didn't check to see how the babel transforms were handled. If you're not polyfilling in `Map`, then I agree it's unacceptable. I'll replace it.